docusaurus/i18n/en/docusaurus-plugin-content-docs-platform/current.json

```diff
@@ -0,0 +1,166 @@
+  },
+  "sidebar.platform.category.Testing Connectors": {
+    "message": "Testing Connectors",
+    "description": "The label for category Testing Connectors in sidebar platform"
+  },
+  "sidebar.platform.category.Deploy Airbyte": {
+    "message": "Deploy Airbyte",
+    "description": "The label for category Deploy Airbyte in sidebar platform"
+  },
+  "sidebar.platform.category.Infrastructure": {
+    "message": "Infrastructure",
+    "description": "The label for category Infrastructure in sidebar platform"
+  },
+  "sidebar.platform.category.Integrations": {
+    "message": "Integrations",
+    "description": "The label for category Integrations in sidebar platform"
+  },
+  "sidebar.platform.category.abctl": {
+    "message": "abctl",
+    "description": "The label for category abctl in sidebar platform"
+  },
+  "sidebar.platform.category.Self-Managed Enterprise": {
+    "message": "Self-Managed Enterprise",
+    "description": "The label for category Self-Managed Enterprise in sidebar platform"
+  },
+  "sidebar.platform.category.Upgrading Airbyte": {
+    "message": "Upgrading Airbyte",
+    "description": "The label for category Upgrading Airbyte in sidebar platform"
+  },
+  "sidebar.platform.category.Configuring Airbyte": {
+    "message": "Configuring Airbyte",
+    "description": "The label for category Configuring Airbyte in sidebar platform"
+  },
+  "sidebar.platform.category.Access Management": {
+    "message": "Access Management",
+    "description": "The label for category Access Management in sidebar platform"
+  },
+  "sidebar.platform.category.Single Sign-On (SSO)": {
+    "message": "Single Sign-On (SSO)",
+    "description": "The label for category Single Sign-On (SSO) in sidebar platform"
+  },
+  "sidebar.platform.category.Role-Based Access Control (RBAC)": {
+    "message": "Role-Based Access Control (RBAC)",
+    "description": "The label for category Role-Based Access Control (RBAC) in sidebar platform"
+  },
+  "sidebar.platform.category.Airbyte at Scale": {
+    "message": "Airbyte at Scale",
+    "description": "The label for category Airbyte at Scale in sidebar platform"
+  },
+  "sidebar.platform.category.Collecting Metrics": {
+    "message": "Collecting Metrics",
+    "description": "The label for category Collecting Metrics in sidebar platform"
+  },
+  "sidebar.platform.category.Security": {
+    "message": "Security",
+    "description": "The label for category Security in sidebar platform"
+  },
+  "sidebar.platform.category.Integrating with Airbyte": {
+    "message": "Integrating with Airbyte",
+    "description": "The label for category Integrating with Airbyte in sidebar platform"
+  },
+  "sidebar.platform.category.Account Management": {
+    "message": "Account Management",
+    "description": "The label for category Account Management in sidebar platform"
+  },
+  "sidebar.platform.category.Understand Airbyte": {
```

Comment:
If we are using i18n for sidebar labels, should we transition labels to a "safer" format in the sidebar definition?
i.e 
```
 const understandingAirbyte = {
  type: "category",
  label: "understand_airbyte",
  items: [
    "understanding-airbyte/high-level-view",
    "understanding-airbyte/airbyte-protocol",
    ...
  ],
};